frontend/src/Student/containers/ViewScholarshipsTab.js

```diff
@@ -0,0 +1,27 @@
+import { compose } from 'recompose'
+import { connect } from 'react-redux'
+import { graphql } from "react-apollo";
+
+import ScholarshipsQuery from "../queries/scholarships.graphql"
+import ViewScholarshipsTab from "../components/ViewScholarshipsTab";
+import { displayError } from "../../shared/actions";
+
+const mapDispatchToProps = dispatch => ({
+  handleError: message => dispatch(displayError(message))
+});
+
+export default compose(
+  connect(null, mapDispatchToProps),
+  graphql(ScholarshipsQuery, {
+    props: ({ ownProps, data: { scholarship, loading, error } }) => ({
+      scholarship,
+      loading,
+      error
+    }),
+    options: ownProps => ({
+      variables: {
```

Comment:
the portfolioId is passed in as a property by the parent component (see frontend/src/Student/pages/Scholarships.js). It is then added to the scholarshipQuery as a query parameter